app.js

```diff
@@ -1,5 +1,58 @@
 /* eslint no-restricted-syntax: [0, "ForInStatement"] */
 
+/**
+ * @swagger
+ * components:
+ *   securitySchemes:
```

Comment:
I think we need to specify that one of the security scheme should be specified on the main route /export/:userid. And because it would accept both user or server token, I would merge those 2 schemes into 1 "tidepoolAuth"